src/Uuid.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Xabbuh\XApi\Model;
+
+final class Uuid
+{
+    private function __construct()
+    {
+    }
+
+    /**
+     * @param string $uuid
+     * @return \Rhumsaa\Uuid\Uuid|\Ramsey\Uuid\Uuid
+     */
+    public static function fromString($uuid)
+    {
+        if (class_exists('Rhumsaa\Uuid\Uuid')) {
+            return \Rhumsaa\Uuid\Uuid::fromString($uuid);
```

Comment:
This method should return an instance of our own `Uuid` class. This way, no information is leaked to the outside about which UUID implementation we use internally.